apps/pie-storybook/stories/testing/pie-toast-provider.test.stories.ts

```diff
@@ -76,24 +76,61 @@ const PositionTemplate = ({ options = defaultProps.options, position }: ToastPro
 };
 
 const CustomZIndexTemplate = () => {
-    const showToast = () => {
+    requestAnimationFrame(() => {
         toaster.create({
-            message: 'This toast should appear above the red box.',
+            message: 'Toast with z-index: 7000 should appear above the overlay.',
             duration: null,
         });
-    };
+    });
 
     return html`
```

Comment:
In general, but especially with AI peer programmers up coming in rapid fashion, I'd recommend to add light comments (e.g. 1 single line comment per 15 lines of code) in new or changed code.

In this case as an example it's not immediately apparent to me why showToast is wrapped in a requestAnimationFrame. Maybe that's obvious to even juniors that join DS teams or perhaps not :)